larray_editor/editor.py

```diff
@@ -634,32 +644,36 @@ def _load_script(self, filepath, lines, symbols):
                 # -s <symbols>: Specify function or classes to load from python source.
                 cmd += ['-s {}'.format(symbols)]
             cmd += [filepath]
+            self.eval_box.input_buffer = '# Press Enter to display the command lines to be loaded and Enter again ' \
```

Comment:
I think we should bypass the load magic to avoid one "enter" (the most confusing one). I think it would be a net win even if we lost the ability to only load some symbols or specific lines.

Reply:
You mean to write our own "load" function?

Reply:
Yes. That should be easy to write if we don't implement the specific lines/symbols feature:
```python
with open(filepath, 'r') as f:
   content = f.read()
self.eval_box.input_buffer = content
```

Reply:
I'm pretty sure users will ask to load only specific lines and/or symbols soon... But OK for the moment.